### PR TITLE
Extract notifications. Accessibility review

### PR DESCRIPTION
**Pull request by ctarda**
Add accessibility hints and labels that were lost while refactoring

To test:
- Check out the branch
- Activate VoiceOver
- Check that the action buttons for notifications provide labels and hints to voiceover

### CODE REVIEW

**Comment by ctarda** [comment]
Thanks for the review @etoledom It turns out the labels and hints in the details view were overwritten in that specific view controller. I have just pushed an update that I think addresses that.

Regarding the Approve button when shipping, I have added a ticket for that: #9746

File: WordPress/Classes/ViewRelated/Notifications/Views/NoteBlockActionsTableViewCell.swift [hunk 1]
```diff
@@ -156,25 +156,25 @@ class NoteBlockActionsTableViewCell: NoteBlockTableViewCell {
...
     fileprivate var approveAccesibilityLabel: String {
-        return isApproveOn ? Approve.selectedTitle : Approve.normalTitle
+        return isApproveOn ? ApproveComment.TitleStrings.selected : ApproveComment.TitleStrings.approve
     }
 
     /// Returns the accessibility hint for the Approve Button
     ///
     fileprivate var approveAccesibilityHint: String {
-        return isApproveOn ? Approve.selectedHint : Approve.normalHint
+        return isApproveOn ? ApproveComment.TitleHints.selected : ApproveComment.TitleStrings.approve
```
Comment (etoledom) [inline WordPress/Classes/ViewRelated/Notifications/Views/NoteBlockActionsTableViewCell.swift]: I think `ApproveComment.TitleStrings.approve` should be `ApproveComment.TitleHints.approve`?
(TitleString -> TitleHints)

**Comment by frosty** [comment]
@etoledom I'd agree with using `the` instead of `a`, because you're talking about a specific comment. 

You mentioned using `deletes` or `removes` the comment – is that to add a bit more explanation than just repeating the term 'trash' again? 

Those sound okay to me if you want to use one of those – or you could do something like `Moves the comment to the trash`.

**Comment by etoledom** [comment]
Thank you @frosty !

I like `Moves the comment to the trash`! It's a good hint to highlight that the comment is not deleted forever, but moved to the "trash folder" (folder that I haven't found in the app btw 😭)

**Comment by ctarda** [comment]
I've just pushed another update. Thanks folks!

**Pull request merged**